### DISCUSSION TITLE
Helping me  with my first open source contribution

### DISCUSSION THREAD

**Discussion by ggps2002** [perspective-dev/perspective#2633] [Q&A] [answered]
I'm fairly new to open source and wish to contribute and be a part of the community. Kindly guide me how to get started

**Comment by ghost** [comment, marked as answer]
Hello! Here are some ways to begin your open-source journey:
1. Github has excellent learning opportunities for using Git, Github, and other tools. One such course is the "The Open Source Education Path" by OpenSauced and Github (https://github.com/open-sauced/intro/blob/main/docs/README.md)
2. Under the explore tab, Check out the "How to choose (and contribute to) your first open source project" collection. Here, there are multiple beginner-friendly opportunities to contribute to various open-source projects. 
3. Also be sure to look for tags like "beginner-friendly", "contibutions-welcome", "good-first-issue", etc. These tags let you know there are opportunities for beginner-level contributions.
